NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -16,11 +16,11 @@ public extension NeuroID {
             return started
         }
        
+        NeuroID.isAdvancedDevice = advancedDeviceSignals
+        
```

Comment:
The start/startSession command should not change the global variable, it should only apply its preference for that specific session/command. 
I.e. start(false) would turn off ADV and they could call startAppFlow that should do ADV
OR
start(true) would do ADV in all flows when they only want it for that one flow

Consider instead passing the variable to this method and then having the method check the parameter